migrations/Migration20240821150800.ts

```diff
@@ -0,0 +1,9 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20240816120248 extends Migration {
```

Comment:
Name mismatch with file (I don't think MikroORM actually cares about the name, but just in case)